docs/contrib/tests.md

```diff
@@ -0,0 +1,26 @@
+We use `pytest`, `flake8` and `mypy` as a quality gate. We follow `wemake_python_styleguide` to enforce quality.
```

Comment:
make `wemake_python_styleguide` a link to wemake site would be nice.
`https://wemake-python-stylegui.de/en/latest/`